.github/workflows/continuous-integration-workflow.yml

```diff
@@ -93,6 +93,7 @@ jobs:
         run: |
           cmake -B builddir \
             -DCMAKE_INSTALL_PREFIX=/usr \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
```

Comment:
Did you mean to fix that here?

Reply:
Yes, I figured that this can be handled in the same PR.

Reply:
Also prefer putting it next to the `DCMAKE_CXX_COMPILER` option

Reply:
> Yes, I figured that this can be handled in the same PR.

OK for this time but move it next to `CMAKE_CXX_COMPILER`

Reply:
Done (7e651ca).